### PR TITLE
Blackbox increase resolution on setpoint, gyro and rounding

### PR DESCRIPTION
**Pull request by daleckystepan**
BF part: https://github.com/betaflight/betaflight/pull/11313

- Increase BB resolution if enabled
- Add GHST RX
- Rename RC Rates to Setpoint

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: js/flightlog_fields_presenter.js
Comment (haslinghuis) [inline js/flightlog_fields_presenter.js:765]:
```suggestion
                return flightLog.gyroRawToDegreesPerSecond(value / 10).toFixed(1) + " deg/s";
```

Comment (haslinghuis) [reply js/flightlog_fields_presenter.js:765]: Same goes for other changes.

Comment (daleckystepan) [reply js/flightlog_fields_presenter.js]: Fixed

**Comment by daleckystepan** [comment]
Log for testing with `set blackboc_high_resolution = ON`
[btfl_004.bbl.zip](https://github.com/betaflight/blackbox-log-viewer/files/7936952/btfl_004.bbl.zip)

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
